studio/localStores/storageExplorer/StorageExplorerStore.js

```diff
@@ -8,7 +8,6 @@ import { createClient } from '@supabase/supabase-js'
 import { useStore } from 'hooks'
 import { copyToClipboard } from 'lib/helpers'
 import { IS_PLATFORM } from 'lib/constants'
-import { PROJECT_ENDPOINT_PROTOCOL } from 'pages/api/constants'
```

Comment:
@ruggi99 It was imported before https://github.com/supabase/supabase/pull/11740#discussion_r1083419924

You can revert this delete.

Reply:
Yeah I know, but the whole point is that `PROJECT_ENDPOINT_PROTOCOL` is always "http" because it is client-side code.
But I will revert it so we can move forward

Reply:
Thanks for helping with this change. Although the value is always http, it's a good practice to move string constants to one place instead of having them scattered in different files.

Reply:
Sounds good from this perspective